postgresql-s3-backup/README.md

```diff
@@ -0,0 +1,22 @@
+Creation of the Cronjob
+- the cronjob is create to make regular backups once at between 00 : 00 and 01 : 00.
+- it is created to make buckups of postgres database data.
+- integrated with amazon-s3 bucket
+- docker image that we used
+- https://github.com/schickling/dockerfiles/tree/master/postgres-backup-s3
+
+Valid parameters for postgres-s3-backup
+Parameter                 | Values
+---------------------------------------------------------------------------------------------------------------------------------------------------------------- 
+- schedule                | '@daily'  *** set the schedule for the cronjob to run at e.g ' 0 0 */1 * *' will run every day at 00 : 00
+- s3_region               | region *** which region is the S3 bucket set
```

Comment:
These should not have underscores. They need to match the yaml path. s3.region, postgres.user